core/client/src/main/java/alluxio/client/file/FileOutStream.java

```diff
@@ -116,7 +120,8 @@ public FileOutStream(AlluxioURI path, OutStreamOptions options) throws IOExcepti
         String tmpPath = PathUtils.temporaryFileName(mNonce, mUfsPath);
         UnderFileSystem ufs = UnderFileSystem.get(tmpPath, ClientContext.getConf());
         // TODO(jiri): Implement collection of temporary files left behind by dead clients.
-        mUnderStorageOutputStream = ufs.create(tmpPath, (int) mBlockSize);
+        CreateOptions createOptions = new CreateOptions().setPermission(options.getPermission());
```

Comment:
Because `CreateUfsFileOptions` and `CompleteUfsFileOptions` have to be converted to Thrift, so its better to keep *Option and *TOption same. While the UFS options here are not converted to Thrift options, so I used a single `Permission` object, which is cleaner and consistent with that in `InodeTree`